packages/react-menu/jest.config.js

```diff
@@ -1,9 +1,18 @@
-const { createConfig, resolveMergeStylesSerializer } = require('@fluentui/scripts/jest/jest-resources');
-const path = require('path');
-
-const config = createConfig({
-  setupFiles: [path.resolve(path.join(__dirname, 'config', 'tests.js'))],
-  snapshotSerializers: [resolveMergeStylesSerializer()],
-});
-
-module.exports = config;
+/**
+ * @type {jest.InitialOptions}
+ */
+module.exports = {
+  displayName: 'react-menu',
+  preset: '../../jest.preset.js',
+  globals: {
+    'ts-jest': {
+      tsConfig: '<rootDir>/tsconfig.json',
+      diagnostics: false,
+    },
+  },
+  transform: {
+    '^.+\\.tsx?$': 'ts-jest',
+  },
+  coverageDirectory: '../../coverage/packages/react-button',
```

Comment:
Is this intentional?

Reply:
The main part I was confused about was why react-menu coverage data appears to be going under a react-button folder (looks like a typo). 

To the other question, I think for now I'd prefer to keep coverage and such under the package folder (maybe under `temp/coverage` so it automatically gets excluded from publishing?) and consider moving things to the root all-up separately. I can see how having everything at the root would simplify some aspects of publishing, but it's a pretty drastic change from both a tooling and contributor expectations standpoint.